adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -19,7 +19,7 @@ package org.bdgenomics.adam.rdd.read
 
 import java.io.StringWriter
 
-import htsjdk.samtools.{ SAMFileHeader, SAMTextHeaderCodec, SAMTextWriter, ValidationStringency }
+import htsjdk.samtools.{ TextCigarCodec, SAMFileHeader, SAMTextHeaderCodec, ValidationStringency, SAMTextWriter }
```

Comment:
I actually agree with you RE: not wildcard importing packages. My personal preference would be to newline it at the commas though, and to alphabetize the classes that are imported (I prefer to keep the imports alphabetized, because it generally keeps diffs smaller).